.gitignore

```diff
@@ -11,3 +11,4 @@
 
 .idea/
 .vscode/
+vendor
```

Comment:
Why adds this line?

Reply:
I run command `go mod vendor` in this project by accident and found that this project doesn't ignore this directory.

Reply:
But no directory named as "vendor" exists in this repo.

Reply:
In Antrea, we also ignore this directory.